lib/client-js/campusEvents.js

```diff
@@ -125,20 +125,20 @@ function insertEvents(eventArray, eventNumber) {
             else {
                 var nameFromJSON = '';
                 var letter;
-                for (var b = 0; b < campus.buildings.length; b++) {
-                    for (var f = 0; f < campus.buildings[b].floors.length; f++) {
-                        for (var n = 0; n < campus.buildings[b].floors[f].roomNames.length; n++) {
-                            if (splitLocations.length >= 2 && campus.buildings[b].floors[f].roomNames[n].name.toLowerCase().includes(splitLocations[0].toLowerCase() + "-" + splitLocations[1].toLowerCase()) == true) {
+                for (const element of campus.buildings) {
+                    for (var f = 0; f < element.floors.length; f++) {
+                        for (let n = 0; n < element.floors[f].roomNames.length; n++) {
+                            if (splitLocations.length >= 2 && element.floors[f].roomNames[n].name.toLowerCase().includes(splitLocations[0].toLowerCase() + "-" + splitLocations[1].toLowerCase()) == true) {
```

Comment:
Another unnecessary change. Why didn't you make them all consistent though. ie 
```
for (const building of campus.buildings) {
  for (const floor of building.floors) {
    for (const room of floor) {
```

That being said, I hope to improve how this is implemented in the future. This code chunk is kinda gross.